verilog/analysis/symbol_table_test.cc

```diff
@@ -1469,6 +1469,23 @@ TEST(BuildSymbolTableTest, ModuleInstanceNamedParameterAssignment) {
   }
 }
 
+TEST(BuildSymbolTableTest, ModuleInstanceThatIsATimerRegressionIssue917) {
```

Comment:
Maybe name this something with `KeywordAsIdentifier` in it to be more descriptive.

Reply:
Done.